src/components/Atoms/Button/Button.test.js

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
-import 'jest-styled-components';
-import renderWithTheme from '../../../hoc/shallowWithTheme';
-import Button from './Button';
+import React from "react";
+import "jest-styled-components";
+import renderWithTheme from "../../../hoc/shallowWithTheme";
+import Button from "./Button";
 
-it('renders a standard styled link correctly', () => {
+it("renders a standard styled link correctly", () => {
```

Comment:
Don't worry about Linting these auto-generated files; it'll just get overridden again next time someone updates the component